src/modules/reporting/components/reporting-report/reporting-report.jsx

```diff
@@ -103,8 +110,26 @@ export default class ReportingReport extends Component {
             history={p.history}
             cardStyle="single-card"
             buttonText="View"
+            showAdditionalDetailsToggle={s.showToggleDetails}
+            showingDetails={s.showingDetails}
+            toggleDetails={this.toggleDetails}
           />
         }
+        { !isEmpty(p.market) && s.showingDetails &&
+          <div className={Styles[`ReportingReportMarket__details-container-wrapper`]}>
+            <div className={Styles[`ReportingReportMarket__details-container`]}>
+              <div className={Styles.ReportingReportMarket__details}>
+                <span>
+                  {p.market.extraInfo}
+                </span>
+              </div>
+              <div className={Styles[`ReportingReportMarket__resolution-source`]}>
+                <h4>Resolution Source:</h4>
+                <span>{p.market.resolutionSource || 'Outcome will be determined by news media'}</span>
```

Comment:
Should this be hard coded?

Reply:
Yes, there are only 2 options when creating a market. The market creator can fill in their website url or say the source will be determined by news media.